microsite/data/plugins/puppetdb.yaml

```diff
@@ -0,0 +1,13 @@
+---
+title: PuppetDB
+author: TDabasinskas
+authorUrl: https://github.com/tdabasinskas
+category: Configuration Management
+description: Visualize resource information and Puppet facts from PuppetDB.
+documentation: https://github.com/backstage/backstage/blob/master/plugins/puppetdb/README.md
+iconUrl: https://avatars.githubusercontent.com/u/234268?s=200&v=4
```

Comment:
Just so we have it some where but where does this come from? It does include the Puppet logo and I'm not sure the rules on it's usage. We might need to change this

Reply:
I've updated it to point to the one listed on Wikipedia under Creative Commons - https://commons.wikimedia.org/wiki/File:Puppet_transparent_logo.svg. Should it be OK?

Reply:
Yeah, I'll be honest I don't know. I just mention it as it's come up in the past. I'll leave this to the maintainers to decide on.

Reply:
But you can put the image in `/microsite/static/img`

Reply:
Added the image to the mentioned path.